style.css

```diff
@@ -0,0 +1,184 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+
+body {
+  overflow: hidden;
+  background: var(--shadow-color);
+  background-attachment: fixed;
+  height: 100vh;
+}
+
+/* Colors */
+:root {
+  --primary-color: white;
+  --secondary-color: #292929;
+  --tertiary-color: black;
+  --shadow-color: #e3e3e3;
+  --selected-color: #0d6efd;
+}
+
```

Comment:
Definitely want to set your variables first in this file. I would recommend moving it up to be the first thing in this file.